### PR TITLE
Update go swag v1.8.4 -> v1.16.2

### LINKED ISSUE #59: swag 1.8.8 breaks Docker container
With the latest release ([v1.8.8](https://github.com/swaggo/swag/releases/tag/v1.8.8)) of [swag](https://github.com/swaggo/swag) the Docker container fails to build with the following error:

```
~/go/bin/swag init -g cmd/query/main.go --md docs
2022/11/28 14:27:44 Generate swagger docs....
2022/11/28 14:27:44 Generate general API Info, search dir:./
2022/11/28 14:27:44 warning: failed to get package name in dir: ./, error: execute go list command, exit status 1, stdout:, stderr:no Go files in /Users/aej/projects/vds-slice
2022/11/28 14:27:44 ParseComment error in file /Users/aej/projects/vds-slice/api/query.go :cannot find type definition: vds.Metadata
```

I tested `swag@v1.8.7` for which the build of the documentation also shows a warning (same as for `swag@v1.8.8`), but is still built without error:

```
2022/11/28 14:40:15 Generate swagger docs....
2022/11/28 14:40:15 Generate general API Info, search dir:./
2022/11/28 14:40:15 warning: failed to get package name in dir: ./, error: execute go list command, exit status 1, stdout:, stderr:no Go files in /Users/aej/projects/vds-slice
2022/11/28 14:40:15 Generating vds.Metadata
2022/11/28 14:40:15 Generating vds.BoundingBox
2022/11/28 14:40:15 Generating vds.Axis
2022/11/28 14:40:15 Generating api.ErrorResponse
2022/11/28 14:40:15 Generating api.MetadataRequest
2022/11/28 14:40:15 Generating api.BaseRequest
2022/11/28 14:40:15 Generating vds.SliceMetadata
2022/11/28 14:40:15 Generating api.SliceRequest
2022/11/28 14:40:15 Generating vds.FenceMetadata
2022/11/28 14:40:15 Generating api.FenceRequest
2022/11/28 14:40:15 create docs.go at  docs/docs.go
2022/11/28 14:40:15 create swagger.json at  docs/swagger.json
2022/11/28 14:40:15 create swagger.yaml at  docs/swagger.yaml
``` 

Should we pin the version in the Dockerfile to `v1.8.7` or should we rather fix the what creates the issue with swagger `v1.8.8`? I am unsure if this error is an issue with swag or with our package. I did not find any issue pointing to this problem in the swag repository. I can also reproduce the error locally on MacOS.

### PR DESCRIPTION
**Pull request by AdilKhurshid**
Updated go swag v1.8.4 -> v1.16.2 in the docker file.

In the latest version of Go SWAG introduced two additional parameters, namely `LeftDelim` and `RightDelim`, which were added to the Swaginfo structure to address issues related to special characters within the doc string.
[ref](https://github.com/swaggo/swag/commit/01d20cc48ca1e07b5510585286b6f0e796714502)

In addition, added flage `--parseDependency` in the doc genrating command in the docker file which parse go files inside dependency folder. By default, this flag is set to false in Go SWAG.

closes #59

### CODE REVIEW

**Review by achaikou — COMMENTED** [review]
So the issue was created when swag updated to version 1.8.8 and suddenly our code, which worked for 1.8.7, stopped working. That didn't seem like intentional breaking change, so we decided to wait and see if someone will raise a bug. So if we changed anything too hastly there was a danger we would need to revert it back.

Erlend/Alex found the first way to deal with it (remove prefix), Yngve found the second (adding go file to root repo), you found the third one (enabling dependencies), I now found the 4th one.

First broke docs for swag 1.8.7, second requires changing file structure, third implies parsing external dependencies.

What I now discovered seems to work for all versions but needs explicitly providing directories for parsing, which is also somewhat annoying as it would break with naming changes
`swag init --dir cmd/query,api,internal/core -g main.go [and all other stuff there is in the command]`

So the main question here is how we want to deal with this situation.

One option is getting a courageous hero (who I am definitely not :D ) to ask a very carefully crafted question to the swag repo regarding what happened between 1.8.7 and 1.8.8 and what is expected way to deal with our issue. I suspect it might be difficult to explain our issue shortly and properly and most likely it would be quite confusing for all sides.

Other option is choose the best solution from the ones we could think of.

Not sure what to do about it.

File: Dockerfile [hunk 1]
```diff
@@ -43,8 +43,8 @@ COPY . .
 ARG CGO_CPPFLAGS="-I/open-vds/Dist/OpenVDS/include"
 ARG CGO_LDFLAGS="-L/open-vds/Dist/OpenVDS/lib"
 RUN go build -a ./...
-RUN GOBIN=/tools go install github.com/swaggo/swag/cmd/swag@v1.8.4
-RUN /tools/swag init -g cmd/query/main.go --md docs
+RUN GOBIN=/tools go install github.com/swaggo/swag/cmd/swag@v1.16.2
```
[the remark this answers is not in the document]
Comment (yngve793) [reply Dockerfile:46]: Pinning the version ensure that the code will run consistently. If it is not pinned we must be prepared that the build will break even if the code is not changed. Pushing updates of packages blindly to servers requires a  lot of trust in the packages developers. What is Equinor´s policy on this?

[the remark this answers is not in the document]
Comment (achaikou) [reply Dockerfile:46]: I don't know if we have a common policy on this. Also don't know if there is something AppSec recommends.

File: Dockerfile [hunk 2]
```diff
@@ -43,8 +43,8 @@ COPY . .
...
+RUN /tools/swag init --parseDependency -g cmd/query/main.go --md docs
```
[the remark this answers is not in the document]
Comment (AdilKhurshid) [reply Dockerfile]: Yes you are right, changing this with the command you mentioned above.

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (AdilKhurshid) [reply Dockerfile:46]: I totally agree with Yngve. It's important to stick with a specific version that we know works well perfectly with our use case. We can periodically update this version to incorporate the latest changes and features from Go SWAG.

**Review by achaikou — COMMENTED** [review]
Looks good to me! :cat: 
Rebase and I'll approve.

**Review by achaikou — APPROVED** [review]

**Pull request merged**